plugins/org.jkiss.dbeaver.ext.ocient/src/org/jkiss/dbeaver/ext/ocient/model/plan/OcientExecutionPlan.java

```diff
@@ -64,7 +64,8 @@ public String getQueryString() {
 
     @Override
     public String getPlanQueryString() {
-        return "explain json" + query;
+        // Adds "explain json" to front of query with proper spacing
+        return String.format("explain json %s", query);
```

Comment:
I don't really think that formatting is required - just use concatenation here.

Reply:
We used to have concatenation, but someone removed the trailing whitespace. This way, it is obvious that the whitespace is intended @ShadelessFox

Reply:
It's obvious the other way around, too.

Reply:
wdym? it wasn't obvious enough clearly if it was removed unintentionally. See https://github.com/Xeograph/dbeaver/commit/41745c344fdfc07bc36547342352d765524c758f